### PR TITLE
Add getIndexes to prototype

### LINKED ISSUE #2: [feature request] Wouldn't a getIndexes method be kind of nice?
Raw Mongo has one. Sometimes I just want to see the indexes for a collection.

### PR DESCRIPTION
**Pull request by menewman**
Fixes #2.

Adds `getIndexes`! Should increment the minor version (or just release version 1.0.0) after this.

### CODE REVIEW

File: src/mongo.js [hunk 1]
```diff
@@ -60,6 +60,18 @@ function extend(Mongo: Object) {
       });
     },
 
+    getIndexes() {
+      return new Promise((resolve, reject) => {
+        this.rawCollection().indexes((error, indexes) => {
+          if (error) {
+            reject(indexes);
+          }
```
Comment (Albinutte) [inline src/mongo.js]:
```suggestion
            return;
          }
```
?

Comment (menewman) [reply src/mongo.js]: screen shot 2018-10-16 at 3 46 51 pm

How interesting.

File: src/mongo.js [hunk 2]
```diff
@@ -60,6 +60,19 @@ function extend(Mongo: Object) {
       });
     },
 
+    getIndexes() {
+      return new Promise((resolve, reject) => {
+        this.rawCollection().indexes((error, indexes) => {
+          if (error) {
+            reject(indexes);
```
Comment (Albinutte) [inline src/mongo.js]:
```suggestion
            reject(error);
```
🙃

Comment (Albinutte) [reply src/mongo.js]: I'm so going to abuse this

Comment (menewman) [reply src/mongo.js]: Wow that was careless of me. I don't like this new feature, though; it feels so aggressive.

Comment (Albinutte) [reply src/mongo.js]: Yeah I agree

Comment (Albinutte) [reply src/mongo.js]: On your feelings towards the new feature, that is

**Pull request merged**